currency-converter/tailwind.config.js

```diff
@@ -5,6 +5,9 @@ export default {
     "./src/**/*.{js,ts,jsx,tsx}",
   ],
   theme: {
+      container: {
```

Comment:
are we using it in application? I don't find a reference to this in code anywhere.